webapp/src/components/post_type_jitsi/post_type_jitsi.test.tsx

```diff
@@ -38,7 +38,7 @@ describe('PostTypeJitsi', () => {
             jwt_meeting: true,
             meeting_jwt: 'xxxxxxxxxxxx',
             meeting_topic: 'Test topic',
-            meeting_id: 'test',
+            meeting_id_label: 'test',
```

Comment:
usually in test cases the dummy data we are using should be similar to the field like 
```suggestion
            meeting_id_label: 'test_meeting_id_label',
```
Something like this